src/core/Akka.TestKit/TestKitBase_Receive.cs

```diff
@@ -309,14 +309,14 @@ public async ValueTask<object> ReceiveOneAsync(TimeSpan? max = null, Cancellatio
         public object PeekOne(TimeSpan? max = null, CancellationToken cancellationToken = default)
         {
             return PeekOneAsync(max, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.